class.application-passwords-list-table.php

```diff
@@ -8,10 +8,10 @@ class Application_Passwords_List_Table extends WP_List_Table {
 
 	function get_columns() {
 		return array(
-			'name'      => __( 'Name', 'two-factor' ),
-			'created'   => __( 'Created', 'two-factor' ),
-			'last_used' => __( 'Last Used', 'two-factor' ),
-			'last_ip'   => __( 'Last IP', 'two-factor' ),
+			'name'      => esc_html__( 'Name', 'two-factor' ),
+			'created'   => esc_html__( 'Created', 'two-factor' ),
+			'last_used' => esc_html__( 'Last Used', 'two-factor' ),
+			'last_ip'   => esc_html__( 'Last IP', 'two-factor' ),
```

Comment:
These columns probably shouldn't be escaped here, as you're providing the content, that the list table class can display in many different methods -- for example, as an attribute like so:

https://github.com/WordPress/WordPress/blob/b72f822a69ad634488b5a57aea99f0a78e42f777/wp-admin/includes/class-wp-list-table.php#L1213

Maybe calling `wp_strip_all_tags()` on it, or `wp_kses()` -- but escaping without knowing the usage context here feels wrong.